.github/workflows/build_doc_prs.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   validate:
-    name: build (18.x) # temporary solution to trick branch protection rules
+    name: build (20.x) # temporary solution to trick branch protection rules
```

Comment:
Careful with the name, as we need to change the branch protection rule. Maybe a good opportunity to just use `build`. It's tricky when using matrix because it changes the names and the branch protection rules with it, but we're not using matrix build anymore

Reply:
I've changed to build now, and once merged, I'll update the protection rules.